src/Peachpie.Library.Graphics/PhpGd2.cs

```diff
@@ -667,16 +667,50 @@ public static long imagecolorallocatealpha(PhpResource im, int red, int green, i
         /// <summary>
         /// RGBA values to PHP Color format.
         /// </summary>
-        static long RGBA(long red, long green, long blue, long alpha = 0xff)
+        static long RGBA(long red, long green, long blue, long alpha = 0)
         {
             return (alpha << 24)
-                | ((blue & 0x0000FF) << 16)
+                | ((red & 0x0000FF) << 16)
                 | ((green & 0x0000FF) << 8)
-                | (red & 0x0000FF);
+                | (blue & 0x0000FF); 
         }
 
-        static Rgba32 FromRGB(long color) => new Rgba32((uint)color | 0xff000000u);
-        static Rgba32 FromRGBA(long color) => (color != (long)ColorValues.TRANSPARENT) ? new Rgba32((uint)color) : (Rgba32)Color.Transparent;
+        static Rgba32 FromRGB(long color) => new Rgba32((uint)color & 0x00ffffffu);
```

Comment:
this will produce a transparent color every time, is it correct?

Reply:
fixed it in commit Review fix